readme.md

```diff
@@ -1,54 +1,36 @@
-* `azure-log-forwarder-function` - function app that subscribes to one Event Hub which is in the same location where you deploy azure-log-forwarder; deployment with App Service Plan and minimum `S1 Standard` pricing tier 
-* `azure-log-forwarder-container` - container instances that provides Dynatrace ActiveGate proxy to Dynatrace Infrastructure. This is Azure Container Instance with 1 core, 1.5 GB ram assigned
-* `azure-log-forwarder-vnet` - virtual network to ensure private communication between function and active-gate
-
-You need to deploy azure-log-forwarder in each region you want to pull logs from.
-
-![Architecture](./img/architecture.png)
+# Overview
+ This project provides a mechanism that allows streaming Azure logs from Azure Event Hub into Dynatrace Logs via Azure Function App. It supports both: Azure Resource Logs and Azure Activity Logs.
 
 
+# Prerequisites
+Please refer to [Prerequisites](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd#prerequisites-) Documentation.
+ 
 # Deployment
-Please refer to the instructions:  
-   [Dynatrace log forwarder Documentation](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd)
+Please refer  to [Dynatrace log forwarder](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd) Documentation.
 
 # Viewing Azure logs in Dynatrace UI
- Please [refer to documentation](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd#view-azure-logs).
+Please refer to [documentation](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd#view-azure-logs).
 
 # Self-monitoring
+Please refer to [self-monitoring](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd#self-monitoring-optional)  Documentation.
 
-### Self-monitoring metrics
-Please [refer to documentation](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd#self-monitoring-optional).
-
-#### Azure Authentication
-Managed identity from Azure Active Directory (AD) is used to authenticate to Azure. It allows an app to easily access other Azure AD-protected resources.
-
-To use Managed identity for Function App, after deployment of azure-log-forwarder is finished you need to add a system-assigned identity. 
-In the portal you need to go to Settings section of Function App and select Identity. Within the System assigned tab, switch Status to On.
-One more step is left - in Azure role assignments you need to grant permission **Monitoring Metrics Publisher** to Resource Group where Function App is deployed.
-
-More info here: [How to use managed identities for App Service and Azure Functions](https://docs.microsoft.com/en-us/azure/app-service/overview-managed-identity?tabs=dotnet)
-
-## Pricing
+# Pricing
  Ingested logs will consume DDUs. For more details: 
   - [Azure service monitoring consumption](https://www.dynatrace.com/support/help/reference/monitoring-consumption-calculation/#expand-azure-service-monitoring-consumption-103)
```

Comment:
Why we use Documentation word in some links but not in other ones?

Reply:
https://github.com/dynatrace-oss/dynatrace-azure-log-forwarder/pull/18#discussion_r648216615